src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationService.kt

```diff
@@ -1,23 +1,41 @@
 package uk.gov.justice.digital.hmpps.makerecalldecisionapi.service
 
 import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions.ActiveRecommendation
 import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions.CreateRecommendationRequest
 import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions.CreateRecommendationResponse
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions.RecallType
 import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions.RecommendationResponse
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions.UpdateRecommendationRequest
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions.UpdateRecommendationResponse
 import uk.gov.justice.digital.hmpps.makerecalldecisionapi.exception.NoRecommendationFoundException
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.jpa.entity.Active
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.jpa.entity.RecallTypeOption
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.jpa.entity.Recommendation
 import uk.gov.justice.digital.hmpps.makerecalldecisionapi.jpa.entity.RecommendationEntity
 import uk.gov.justice.digital.hmpps.makerecalldecisionapi.jpa.entity.RecommendationModel
 import uk.gov.justice.digital.hmpps.makerecalldecisionapi.jpa.repository.RecommendationRepository
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
 import kotlin.jvm.optionals.getOrNull
 
 @Service
 internal class RecommendationService(
   val recommendationRepository: RecommendationRepository
 ) {
-  fun createRecommendation(recommendationRequest: CreateRecommendationRequest): CreateRecommendationResponse {
-    val savedRecommendation = recommendationRepository.save(RecommendationEntity(data = RecommendationModel(crn = recommendationRequest.crn)))
+  fun createRecommendation(recommendationRequest: CreateRecommendationRequest, username: String?): CreateRecommendationResponse {
+    val lastModifiedDate = LocalDateTime.now().format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm"))
```

Comment:
Will the date-time be returned in the format `2021-11-05T14:34:23.000Z` as per https://dsdmoj.atlassian.net/wiki/spaces/MRD/pages/4009590838/3.+How+to+store+times+and+adjusting+for+DST ? ie returned as UTC

Reply:
I have formatted it here, so it won't. No problem, I'll remove the formatter.

Reply:
Done :)